packages/flame/lib/src/camera/viewfinder.dart

```diff
@@ -169,8 +169,8 @@ class Viewfinder extends Component
 
   @override
   void onGameResize(Vector2 size) {
-    super.onGameResize(size);
     _initZoom();
+    super.onGameResize(size);
```

Comment:
I am curious on why the change in the order?

Reply:
If you have children of the viewfinder, you might want to know the new zoom from your parent when the child is running `onGameResize`. (Same thing with viewport, where it actually caused some bugs)

Reply:
Maybe we should document that?

Reply:
Doesn't the user expect that the parent has the correct value? Or how do you mean document it? :sweat_smile:

Reply:
I mean adding some kind of docs that if you want the change the zoom on the on game resize, that you might want to call the super after calculating the new values, I mean, now that you explained to me it is pretty obvious, but at first I didn't realized it haha

Reply:
This is the core viewfinder, on a normal component you can do what you want and it will still work, and if you subclass the viewfinder you might want to handle zoom differently and you definitely know what you're doing if you've gone that deep. So I don't think there is anything obvious to document. 😅